newsroom/utils.py

```diff
@@ -200,9 +200,9 @@ def get_agenda_dates(agenda: Dict[str, Any], date_paranthesis: bool = False) ->
 
     if all_day:
         return (
-            "{}".format(date_short(start))
+            "{}".format(start.strftime("%d/%m/%Y"))
```

Comment:
we should use config for the format

Reply:
I mean an existing one https://github.com/superdesk/newsroom-core/blob/develop/newsroom/web/default_settings.py#L315

Reply:
yep, however this configuration is flask-babel specific, and we removed the use of babel from here because we were having timezone issues with it. Also, `strftime` does not accept this format.

Reply:
you can use `flask_babel.format_date` here just make sure to set `rebase=False` there:
https://python-babel.github.io/flask-babel/#flask_babel.format_date

Reply:
Thanks